includes/class-wc-siftscience-element.php

```diff
@@ -108,11 +108,7 @@ public function create( $type, $id, $label = '', $desc = '', $element_options =
 			switch ( $type ) {
 				case self::CUSTOM:
 					$type = 'wc_sift_' . $id; // this is the custom type name needed by WooCommerce.
-					if ( 'anon_id' === $id ) {
-						add_action( 'woocommerce_admin_field_' . $type, array( $this, 'custom_inline_element' ) );
-					} elseif ( 'anon_id1' === $id ) {
-						add_action( 'woocommerce_admin_field_' . $type, array( $this, 'custom_inline_element1' ) );
-					}
+					add_action( 'woocommerce_admin_field_' . $type, array( $this, $element_options['callback'] ) );
```

Comment:
i prefer adding this line after the action for clean up 
unset($element_options['callback']); 
ps: it would be much better to verify array existence prior the second switch after the case 3: line

Reply:
This is just an example/dirty code. For sure these things need to be considered and re-done the right way.